ecommerce/mail_api.py

```diff
@@ -0,0 +1,80 @@
+"""Ecommerce mail functions"""
+import pycountry
+import logging
+
+from mail import api
+from mail.constants import EMAIL_PRODUCT_ORDER_RECEIPT
+
+log = logging.getLogger()
+
+
+def send_ecommerce_order_receipt(order, cyber_source_provided_email=None):
```

Comment:
Could you update this to used `mitol-django-mail` instead? We should do away with the `mail` app so there's not the temptation to use it in new code.